src/configSchema.ts

```diff
@@ -222,6 +223,9 @@ export const VALIDATION_SCHEMA = z
 		return true;
 	})
 	.refine(
-		(config) => config.matchMode === MatchMode.SAFE || config.linkDir,
+		(config) =>
+			(config.matchMode === MatchMode.SAFE &&
+				!config.seasonFromEpisodes) ||
+			config.linkDir,
```

Comment:
```suggestion
		(config) =>
			(config.matchMode === MatchMode.SAFE) &&
				(config.seasonFromEpisodes &&
			!config.linkDir,
```
Can you explain this for me, the logic isn't readily apparent to me on why this isn't grouped differently....

Reply:
This says that linkDir is optional if you are in matchMode safe and seasonFromEpisodes is false.

So basically you need linkDir if you aren't on safe OR you have this feature enabled. The part about matchMode is not for this feature but for risky/partial needing linkDir.